pyccel/codegen/printing/fcode.py

```diff
@@ -1152,10 +1152,7 @@ def _print_AliasAssign(self, expr):
             rhs = rhs.variable
 
         if isinstance(lhs, TupleVariable) and not lhs.is_homogeneous:
-            if isinstance(rhs, (TupleVariable, PythonTuple)):
-                return self._print(CodeBlock([AliasAssign(l, rhs[i]) for i,l in enumerate(lhs)]))
-            else:
-                return self._print(CodeBlock([AliasAssign(l, IndexedVariable(rhs)[i]) for i,l in enumerate(lhs)]))
+            return self._print(CodeBlock([AliasAssign(l, rhs[i]) for i,l in enumerate(lhs)]))
```

Comment:
Is it necessary to index `rhs` explicitly, or can we write this instead?
```
[AliasAssign(l, r) for l, r in zip(lhs, rhs)]
```

Reply:
You are right we can use zip